src/models/validators/test-types/testTypesSchemaSpecialistTestsPut.ts

```diff
@@ -25,6 +26,9 @@ export const testTypesCommonSchemaSpecialistTests = Joi.object()
         }),
       )
       .required(),
+    defects: Joi.array()
+      .items(defectsCommonSchemaSpecialistTestsSubmitted)
+      .optional(),
```

Comment:
Defects is required for light vehicle tests but not included in the payload for other specialist tests, so it's been made optional